dashboard/client/src/pages/task/TaskPage.tsx

```diff
@@ -255,6 +268,16 @@ const TaskLogs = ({
           },
         ] as const)
       : []),
+    ...(task_log_info === null
```

Comment:
Could we return some empty log viewer component with the message? Embedding the message inside what's usually the log viewer is a bit hacky.

If it's reusable, we can do what's done for MultiTabLogViewer:

https://github.com/ray-project/ray/blob/538c075c7546cc3ec373ee0704bd02868b7d95b7/dashboard/client/src/common/MultiTabLogViewer.tsx#L46-L48

Reply:
I'd rather do this as a follow-up for ray 2.6 since this is an existing pattern we use elsewhere. This is a release-blocker/cherry-pick so it should be isolated to only the changes necessary to resolve the issue.